docs/getting-started/developers/index.md

```diff
@@ -179,6 +178,17 @@ Add this to your `index.html`:
 <link rel="stylesheet" href="https://assets.finn.no/pkg/@warp-ds/css/v1/tokens/finn-no.css">
 ```
 
+### Make sure you import the resets and components classes from EIK cdn:
+
+```css
+@import url(https://assets.finn.no/pkg/@warp-ds/css/v1/resets.css);
+@import url(https://assets.finn.no/pkg/@warp-ds/css/v1/components.css);
+```
```

Comment:
Are we sure we should suggest the use of `@import` rule? I remember having a discussion around this having performance issues. [Here's an article](https://www.debugbear.com/blog/avoid-css-import#:~:text=Why%20does%20CSS%20%40import%20slow,can%20download%20them%20in%20parallel) I was shown with regard to this topic.

Reply:
That's a really good point. I'll fix 👍